plugins/optimization-detective/storage/rest-api.php

```diff
@@ -174,7 +174,7 @@ function od_handle_rest_request( WP_REST_Request $request ) {
 		);
 	}
 
-	$data = $request->get_json_params();
+	$data = $request->get_json_params(); // TODO: Why not just get_params()?
```

Comment:
I remember. It's because any URL query parameter would then be included among the params, so by just getting the params from the JSON body we avoid potential failures due to additional query parameters being injected into the request URL.

```suggestion
	$data = $request->get_json_params();
```